modules/nf-core/bcftools/concat/tests/main.nf.test

```diff
@@ -97,7 +97,7 @@ nextflow_process {
             assertAll(
                 { assert process.success },
                 { assert snapshot(
-                    file(process.out.vcf[0][1]).name,
+                    process.out.vcf,
```

Comment:
This is now checking everything, so just process.out?

Reply:
Yeah true, but it's the same now right? :p I'll change that if the tests are failing now (don't want to overdo the CI right now :grin:)